rush.json

```diff
@@ -60,6 +60,12 @@
       "reviewCategory": "production",
       "shouldPublish": true
     },
+    {
+      "packageName": "@autorest/testing",
+      "projectFolder": "packages/libs/testing",
```

Comment:
This isn't really a lib in the sense that we don't ship it as a library, so it could also be in a `packages/testing` folder to go along with the regression testing "package".  Might have to name it `@autorest/test-utils` or something like that though.  If it's not worth the effort, I'm fine with leaving it as-is